setup_resources.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# Load environment variables from .env file
+set -a
+source .env
+set +a
+
+# Set cluster-specific environment variables
+RESOURCEGROUP="v4-westeurope"
```

Comment:
Wouldn't it be better that the Bash file would start with the _RESOURCEGROUP_ variable as an input (and maybe "fall" to `v4-westeurope` value if it was not set)?

Reply:
fix it